.travis.yml

```diff
@@ -4,8 +4,9 @@ python:
 install:
   - pip install flake8
   - pip install mypy
+  - pip install flake8-docstrings
 script:
-  - flake8 *.py **/*.py
+  - flake8 --ignore=D10
```

Comment:
Move this to a config file so everyone uses the same settings - see http://flake8.pycqa.org/en/latest/user/configuration.html